modules/log6x-audit-log-filtering.adoc

```diff
@@ -45,8 +45,10 @@ spec:
     name: <service_account_name>
   pipelines:
     - name: my-pipeline
-      inputRefs: audit # <1>
-      filterRefs: my-policy # <2>
+      inputRefs:
+        - audit # <1>
+      filterRefs:
+        - my-policy # <2>
```

Comment:
If `my-policy` is a user-replaced value, it should be formatted with underscores according to the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values
```suggestion
        - my_policy # <2>
```

Reply:
Hi @lahinson ,
Thank you for reviewing it. I have committed the suggestions. Also this change needs to be applied from 4.14 to 4.17.